client/unit-tests/symbolic/test_static_collection.py

```diff
@@ -0,0 +1,152 @@
+import pytest
```

Comment:
should we update the file name where these tests are stored to match the file names in which these functions are housed?